### DISCUSSION TITLE
Can't find pwospf-all file for MacOS

### DISCUSSION THREAD

**Discussion by choejms** [ANLAB-KAIST/KENSv3#234] [Project Q&A 4. Etc]
This is resolved.

**Comment by Pusnow** [comment]
Did you clone or redownload KENSv3 from GitHub? PWOSPF is added recently.

**Comment by seo-rii** [comment]
If you forked KENS, you may apply this patch:

https://github.com/ANLAB-KAIST/KENSv3/compare/v3.3.5...master.patch
